src/Factory/AbstractFieldsConfigurationFactory.php

```diff
@@ -130,7 +132,19 @@ private function isExcluded(ReflectionMethod $method): bool
      */
     protected function getAnnotationReader(): Reader
     {
-        return $this->entityManager->getConfiguration()->getMetadataDriverImpl()->getReader();
+        $mappingDriver = $this->entityManager->getConfiguration()->getMetadataDriverImpl();
+        if ($mappingDriver instanceof AnnotationDriver) {
+            return $mappingDriver->getReader();
+        }
+        if ($mappingDriver instanceof MappingDriverChain) {
+            foreach ($mappingDriver->getDrivers() as $driver) {
```

Comment:
This only allows annotations on one driver.  Often when using a MappingDriverChain there will be multiple drivers which may each use annotations.